primitives/runtime-interface/proc-macro/src/utils.rs

```diff
@@ -160,3 +214,59 @@ pub fn get_trait_methods<'a>(trait_def: &'a ItemTrait) -> impl Iterator<Item = &
 			_ => None,
 		})
 }
+
+fn parse_version_attribute(version: &Attribute) -> Result<u32> {
+	let meta = version.parse_meta()?;
+
+	let err = Err(Error::new(
+			meta.span(),
+			"Unexpected `version` attribute. The supported format is `#[version(1)]`",
+		)
+	);
+
+	match meta {
+		Meta::List(list) => {
```

Comment:
```suggestion
		Meta::List(list) if list.nested.len() == 1 => {
```

Than we could remove the first branch.

Or doesn't this work because we take the value by value?